### ISSUE TITLE
composer require psr/container fails

### ISSUE THREAD

**Issue by olekhy** [nikolaposa/monolog-factory#10]
Hey and thx for this lib. May u will bump version of psr/container to ^2.0. On try to install psr/container i get follow problem from composer

```
Problem 1
    - nikolaposa/monolog-factory[dev-master, 3.1.0] require psr/container ^1.0 -> found psr/container[1.0.0, 1.1.0, 1.1.1, 1.1.x-dev] but it conflicts with your root composer.json require (^2.0).
    - nikolaposa/monolog-factory 3.1.x-dev is an alias of nikolaposa/monolog-factory dev-master and thus requires it to be installed too.
    - Root composer.json requires nikolaposa/monolog-factory ^3.1 -> satisfiable by nikolaposa/monolog-factory[3.1.0, 3.1.x-dev (alias of dev-master)].

```
thx in advance

**Comment by nikolaposa** [comment]
I'll see what I can do, thanks for reporting this.

**Comment by nikolaposa** [comment]
I've published [new release (3.1.1)](https://github.com/nikolaposa/monolog-factory/releases/tag/3.1.1) that fixes this issue.
